bigtable/filter.go

```diff
@@ -144,6 +144,18 @@ func (lnf latestNFilter) proto() *btpb.RowFilter {
 	return &btpb.RowFilter{Filter: &btpb.RowFilter_CellsPerColumnLimitFilter{CellsPerColumnLimitFilter: int32(lnf)}}
 }
 
+// ApplyLabelFilter returns a filter that applies the
+// given label to all cells in the output row.
+func ApplyLabelFilter(label string) Filter { return applyLabelFilter(label) }
```

Comment:
nit: should this just be LabelFilter? Might match naming of other filters better?

Reply:
cc @tritone thoughts?

Reply:
Hmm the proto naming also has `Apply` in it, but I'm fine either way. @kolea2 @billyjacobson thoughts?

Reply:
Let's make it LabelFilter. StripValueFilter is also an apply type of filter and doesn't start with apply. Good catch

Reply:
Done